todo_app/data/trello_items.py

```diff
@@ -0,0 +1,89 @@
+from flask import session
+import os
+import requests
+import json
+
+
+from dotenv import load_dotenv
+
+
+load_dotenv()
```

Comment:
This shouldn't be necessary. When you run `flask run`, Flask will automatically load the .env file.